homeassistant/components/zha/core/registries.py

```diff
@@ -23,6 +24,7 @@
 
 SMARTTHINGS_HUMIDITY_CLUSTER = 64581
 SMARTTHINGS_ACCELERATION_CLUSTER = 64514
+SMARTTHINGS_ARRIVAL_SENSOR_DEVICE_TYPE = 0x8000
```

Comment:
Let's stick to a consistent number format ?

Reply:
I’ll convert the other 2. Zigbee is generally all in hex